test/cpp/api/clip_grad.cpp

```diff
@@ -0,0 +1,110 @@
+#include <gtest/gtest.h>
+
+#include <torch/torch.h>
+
+#include <test/cpp/api/support.h>
+
+#include <torch/nn/utils/clip_grad.h>
```

Comment:
Thanks for the addition of `torch/csrc/api/include/torch/nn/utils.h` and including it in `torch/csrc/api/include/torch/nn.h`! With that, I believe we don't need to include `torch/nn/utils/clip_grad.h` here, because `#include <torch/torch.h>` should do the work. ;)